### PR TITLE
Feat: add action classes for interleave and index

### PR DESCRIPTION
**Pull request by taka-oyama**
For autocompletion in PhpStorm

### CODE REVIEW

File: src/Schema/IndexDefinition.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Colopl\Spanner\Schema;
+
+use Illuminate\Support\Fluent;
+
+/**
+ * @method $this interleave(string $table)
+ * @method $this nullFiltered()
+ * @method $this storing(string[] $columns)
```
Comment (tyabu12) [inline src/Schema/IndexDefinition.php:10]: Unrelated to this PR, but It might be nice to explain the nullFiltered and storing index in README.md

**Review by tyabu12 — APPROVED** [review]
LGTM

File: src/Schema/IndexDefinition.php (hunk 1 above)
Comment (taka-oyama) [reply src/Schema/IndexDefinition.php:10]: docs have been added.

Comment (tyabu12) [reply src/Schema/IndexDefinition.php:10]: Perfect, thank you.

**Pull request merged**